Userland/Libraries/LibCrypto/Curves/Ed25519.cpp

```diff
@@ -281,7 +281,7 @@ void Ed25519::multiply(u8* result_low, u8* result_high, u8 const* a, u8 const* b
     u32 temp = 0;
     for (u32 i = 0; i < n; i++) {
         for (u32 j = 0; j <= i; j++) {
-            temp += (uint16_t)a[j] * b[i - j];
+            temp += a[j] * b[i - j];
```

Comment:
Wouldn't this overflow u8? The u16 cast should probably stay (in the form of a static_cast<u16> that is)

Reply:
No, arithmetic operations are implicitly promoted to u32 here. :) See https://godbolt.org/z/aqqahazdq